modules/pipeline/pipengine/actionexecutor/plugins/scheduler/scheduler.go

```diff
@@ -386,6 +386,7 @@ func (s *Sched) Cancel(ctx context.Context, action *spec.PipelineTask) (data int
 	}
 	if !shouldDispatch {
 		logrus.Infof("task executor %s execute cancel", taskExecutor.Name())
+		action.Extra.UUID = task_uuid.MakeJobID(action)
```

Comment:
How about other actionexecutors?

Reply:
move makeJobID to framework may change task uuid in database

Reply:
put it in the comment.